### ISSUE TITLE
Single Click Doesn't work on Long names in Chart Summary View

### ISSUE THREAD

**Issue by nitin10s** [aboutcode-org/scancode-workbench#444]
### Description
Refer to the video below, I observed when the filter name is too long, single click that generally opens the Table View doesn't work. On clicking twice or thrice, the Table View opens up.

![Single-Click-Doesn_t-Work](https://user-images.githubusercontent.com/16988558/79586001-318b3580-80ee-11ea-8396-094367a31fe2.gif)

**Comment by steven-esser** [comment]
This is a similar (if not the same) issue as #443. This is a very common occurence throughout the workbench and is tracked in #334

**Comment by nitin10s** [comment]
No, I think it's different. Here I mean, due to the long text, the single click functionality doesn't seem to work, so we need to double (sometimes triple) click for opening the Table View.

**Comment by steven-esser** [comment]
@nitin10s I see more clearly now. This is definitely some sort of UI bug.

**Comment by devil-cyber** [comment]
Hey can I work on this issue

**Comment by steven-esser** [comment]
@devil-cyber Sure :)
